commit a06b1414a4d1850244827dc61382f44525151e92
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Wed Sep 29 18:07:31 2010 -0700

    Staging: brcm80211: remove Module.symvers file from the tree
    
    This shouldn't have been in the tree, it's generated by the build, so
    remove it.
    
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

diff --git a/drivers/staging/brcm80211/Module.symvers b/drivers/staging/brcm80211/Module.symvers
deleted file mode 100644
index e69de29..0000000
